pip/commands/run.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import absolute_import
+
+from pip.basecommand import Command, SUCCESS
+
+from pip._vendor import rwt
+
+
+class RunCommand(Command):
+    """Show help for commands"""
```

Comment:
That docstring looks incomplete.

More generally, does this patch include suitable documentation for ```pip run```? I know a lot of the docs are auto-generated, but given that this command just defers all its processing to rwt, I guess the auto-generation won't work. Can you add some documentation so that the full syntax for the command (and some examples) show up in the pip documentation?